_posts/2020-05-04-introducing-open-sgid.md

```diff
@@ -12,15 +12,15 @@ tags:
 
 <div class="flex flex--center pop" style="max-width:50%;justify-content:center">
   <h4 style="margin-bottom:0;padding-top:0"><strong>Database Platform</strong>: PostgresSQL</h4>
   <h4 style="margin-top:0;margin-bottom:0;padding-top:0"><strong>Instance</strong>: opensgid.agrc.utah.gov</h4>
   <h4 style="margin-top:0;margin-bottom:0;padding-top:0"><strong>Username</strong>: agrc <strong>Password</strong>: agrc</h4>
 </div>
 
-The timing could not be more perfect with the sudden increase in teleworking, which is not expected to wane any time soon, especially here in Utah with the implementation of the [Governor’s teleworking initiative](https://governor.utah.gov/2019/07/16/state-of-utah-introduces-teleworking-program-for-employees/). Not only does the Open SGID benefit Utahns, but it also signals the return to a truly publicly-available, direct database connection that can be utilized on a multitude of GIS platforms. Furthermore, it initiates the sunsetting of the state network-only SGID10 database, although both databases (Open SGID and SGID10) will run in parallel for some time to allow users to make the transition to the Open SGID and adapt to the change.
+The timing could not be more perfect with the sudden increase in teleworking, which is not expected to wane any time soon, especially here in Utah with the implementation of the [Governor’s teleworking initiative](https://governor.utah.gov/2019/07/16/state-of-utah-introduces-teleworking-program-for-employees/). Not only does the Open SGID benefit Utahns, but it also signals the return to a truly publicly available, direct database connection that can be utilized on a multitude of GIS platforms. Furthermore, it initiates the sunsetting of the state network-only SGID10 database, although both databases (i.e., the Open SGID and the SGID10) will run in parallel for some time to allow users to transition and adapt to the Open SGID.
 
 The new tools and processes developed collectively by AGRC staff to accomplish the Open SGID, as well as the new and improved [Open Data site](https://opendata.gis.utah.gov/), are numerous and worthy of their own blogpost - perhaps some foreshadowing there. However, it is worth noting that a paradigm shift led to the creation of a new internal SGID database that serves as the sole editing environment and data source for the Open SGID and the Open Data site. Automated processes provide nightly updates to both data resources to maintain consistency between them, so you can be sure that you are utilizing the most current data available.
```

Comment:
```suggestion
The new tools and processes developed by AGRC staff to accomplish the Open SGID, as well as the new and improved [Open Data site](https://opendata.gis.utah.gov/), are numerous and worthy of their own blogpost---perhaps some foreshadowing there. In this post though, we'll note that a paradigm shift led to the creation of a new internal SGID database that serves as the sole editing environment and data source for the Open SGID and the Open Data site. Automated processes provide nightly updates to both data resources to maintain consistency between them, so you can be sure that you are utilizing the most current data available.
```

Reply:
Also, I'm wondering about the "accomplish" in the first sentence--Do we mean "develop" or something like that? Or maybe "manage"? etc.

Reply:
Yes, I did mean "develop", but thought it came too closely on the heels of the previous "developed" in the sentence. Maybe the first one can be changed to "built", or something like that?